Vanderlin Map | Map changes to the keep and garrison.

CannibalHunter opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- Note: PRs including balance changes authored by anyone other than maintainers and official devs will not be considered. -->

## About The Pull Request
Moves the garrison to the bridge, they have gate that spawns open.
All the doors in the keep and garrison use the mapping helpers to assign them lockid and locked.
Change the captains lock id and spawnpoint from sherrif to captain.
Puts the captains office in the keep.
Moves the squire/knight spawnpoints to be in thier bedrooms.
Touches up a lot of the rocky terrain.
Changes up the forest garrison hut to be slightly different.

![425727275-6e889b62-11d5-4ce1-84b2-bd0b3815100f](https://github.com/user-attachments/assets/6e135852-eb4d-4f8f-a073-74aa2447dd92)

![425727289-677bb142-251e-4490-958a-6bbcd1103113](https://github.com/user-attachments/assets/d3a73378-3d64-4656-9575-128d07f85357)

![425727351-6555a84d-d46c-471e-bf24-4226a4d5d3f1](https://github.com/user-attachments/assets/d51249ed-0040-44dc-8d60-fd7dddee194b)

![425727514-ee37aad1-df5f-4c4c-9f2f-bf4260a397d9](https://github.com/user-attachments/assets/6bdf29df-9931-49ee-9e49-d4c614a3e094)

## Why It's Good For The Game
Captain having their own room is pretty nice, they didn't have one. As they just spawned in the old garrison and most of the time just went off to the keep.

The Garrison being in a more IN YOUR FACE location helps people find it. Also the new layout has more space and a better armory that has spare helmets, and other things.

Starting the use of door mapping helpers is great.

## Pre-Merge Checklist
<!-- Don't bother filling these in while creating your Pull Request, just click the checkboxes after the Pull Request is opened and you are redirected to the page. -->
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.
<!-- Neither the compiler nor workflow checks are perfect at detecting runtimes and errors. It is important to test your code/feature/fix locally. -->

Sadboysuss commented:
conflicts

Sadboysuss commented:
forest garrison key doesn't seem to open forest garrison

Sadboysuss commented:
![image](https://github.com/user-attachments/assets/6c563212-3851-44aa-80d5-cf42c8f9c680)

doors into the garrison are transparent?

Sadboysuss commented:
top most barred table in the gate is unreachable by the gate guard

CannibalHunter commented:
@Sadboysuss 
> forest garrison key doesn't seem to open forest garrison
Fixed this.
> doors into the garrison are transparent?
Intended, that door type is a window.
> top most barred table in the gate is unreachable by the gate guard
Table has been removed and replaced with a window

wgtjunior743 commented:
I am not sure if the update did not went through, but as a forest guard i still can't open any of the doors

CannibalHunter commented:
@wgtjunior743 
> I am not sure if the update did not went through, but as a forest guard i still can't open any of the doors
Updates on the server are not automatic.

wgtjunior743 commented:
> @wgtjunior743
> 
> > I am not sure if the update did not went through, but as a forest guard i still can't open any of the doors
> 
> Updates on the server are not automatic.

yes that what i meant when i  said "if the update did not went through", probably will be alright in the next round
